roles/openshift_cluster_monitoring_operator/templates/cluster-monitoring-operator-config.j2

```diff
@@ -65,3 +65,6 @@ data:
       baseImage: {{ openshift_cluster_monitoring_operator_kube_rbac_proxy_image }}
     auth:
       baseImage: {{ openshift_cluster_monitoring_operator_proxy_image }}
+    http:
+      httpProxy: {{ openshift.common.http_proxy  | default('') }}
+      httpsProxy: {{ openshift.common.https_proxy  | default('') }}
```

Comment:
deployment seems to work fine with this, but those settings bubble down to the operator even if there is no value set. I will change this here to be a conditional.

Reply:
ok, rethinking: it doesn't make a difference as we set empty http proxy env variables in CMO anyways. This can stay as-is and keeps things a bit simpler too.